src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -31,14 +32,21 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(@RequestParam Authentication authentication,
```

Comment:
why @RequestParam?
should Authentication go as first param in this method or it should go second?
let's test it